docs/en/guides/build-device/image-building.md

```diff
@@ -125,16 +145,20 @@ Then, create the image:
 
 You can include specific snaps pre-installed by default in the image by using the `--extra-snaps` argument. For example:
 
-    sudo ubuntu-image -c beta --extra-snaps rocketchat-server --extra-snaps nextcloud -o pi3-test.img pi3.model
+```bash
+sudo ubuntu-image -c beta --extra-snaps rocketchat-server --extra-snaps nextcloud -o pi3-test.img pi3.model
+```
 
 Note: The `--extra-snaps` argument takes either a snap name accessible from the store or a local path to a built snap.
 
 ### 4. Your image is ready
 
-You can use a tool like `dd` to write the image to an SDcard and boot your device.
+You can use a tool like `dd` to write the image to an SD card and boot your device (replace *sdXX* with the node for your storage device):
 
-    sudo dd if=pi3-test.img of=/dev/sdb bs=32M
-    sync
+```bash
+sudo dd if=pi3-test/pi3.img of=/dev/sdXX bs=32M
```

Comment:
We *might* mentin godd here - but then this needs to get installed separately so maybe too much given that dd is available everywhere.

Reply:
Yeah. The `dd` example was already in the document - I removed the sdb device node in case someone simply pasted the command. But I think it works as an illustration of how the image file can now be written to your SD card - it's highly probable people following this doc already know and have their own preferences for this step.